plugin/someteam.example.com/v1/validator/Validator

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+temp_file=$(mktemp)
+output_file=$(mktemp)
+
+cat - >& $temp_file
+
+dir=$(dirname "${BASH_SOURCE[0]}")
+$dir/kubeval $temp_file >& $output_file
```

Comment:
Let's just assume that kubeval is on the $PATH

We shouldn't check binaries into github.

To be fancy, the location of kubeval from the plugin config yaml, and default it to `kubeval`, as done for `helm` in
https://github.com/kubernetes-sigs/kustomize/blob/master/plugin/someteam.example.com/v1/chartinflator/ChartInflator

We'll have to suppress this test on travis too,
in the same way that the helm test is suppressed.
```
// +build notravis
```
See also `testNoTravisGoTest` in bin/pre-commit.sh.

Later a fancier validator plugin could depend on kubeval packages directly.

Reply:
updated and also added to the no Travis test list.